### DISCUSSION TITLE
How often does OSM maps update for OsmAnd?

### DISCUSSION THREAD

**Discussion by sargenthp** [osmandapp/OsmAnd#19180] [Maps Related]
I have been updating OSM for my area.  I am still not seeing the updates in OsmAnd, and some changes I have made almost a week ago.  When I try to redownload or refresh the map, the date on the map shows the end of January.

**Comment by mikehgentry** [comment]
https://osmand.net/docs/user/personal/maps/#updates

**Reply by sargenthp** [reply]
Thank you.  Missed that part.  I am unable to do a search on the added addresses still.  Am I missing something else?

Thank you!

**Reply by mikehgentry** [reply]
If the maps haven't updated, and you don't have OsmAnd Live, then I wouldn't expect it to work for any recent additions (there's an online search under 'Categories', which I assume updates fairly rapidly, but I don't actually know).

Otherwise I wouldn't expect to be able to find specific addresses easily (that's always been erratic. Changing the order of tokens can help https://github.com/osmandapp/OsmAnd/issues/18924 ). Finding roads / businesses normally works fine for me though. Perhaps it'd be helpful if you gave a specific example of what you're not finding.

**Reply by sargenthp** [reply]
Ok.  I did enable Live.

Let's pick on... 1010 North Town Hall Road , Eau Claire, WI 54703

**Reply by mikehgentry** [reply]
Well it shows up in the 'Online search'. I'd try playing around with the way it's tokenised - maybe put a comma after the 1010, maybe put the 1010 after the road name as per the above thread. The 'Address' search tab might make it easier.

**Reply by tommycrock** [reply]
Sometimes the live updates don't seem to get properly indexed and a newly added address only appears in search results when the next monthly download appears, or that's what it seems like to me.

**Reply by sargenthp** [reply]
I wondered.  I will see what happens at the turn of the month.  

Thanks!

**Comment by The-MaddHatter** [comment]
Just to add correct info for when this thread comes up in search's as it did for me on live updates not containing recent edits even weeks old.

If you go on OSMAND support page under "Maps & Resources".  Then scroll way down to the "Live Updates" section. Go to the bottom of that section right  above  "Free for OSM Mapper" section you will find "Limitations if Live Update".  Here is a link to [Live Updates](https://osmand.net/docs/user/personal/maps-resources/#live-updates). 

Scroll to  Limitations of Live Updates.  Here is it quoted:


>Limitations of Live Updates
**_Address changes are supported only via monthly updates_**.
Updated roads lack elevation details, affecting gradient graphs. Deleted or altered route references may still appear incorrectly.


So yes you pay for live updates but get nothing better than what free version users get for the most updated and missing data on OSM maps. Yet OSM has the data live almost in real time

**Comment by thpar** [comment]
Could it be that also turn restrictions and driving directions are not included in the Live Updates?

My neighborhood has been reorganized last Monday. On Tuesday, the city traffic department already appears to have added these changes to OSM. But although I'm paying for live updates on OsmAnd, I still was completely stuck yesterday with the OsmAnd routing not reflecting the actual situation anymore.

This is quite essential information for eg. taxi or delivery drivers...
